### ISSUE TITLE
Creating a new project results in two build pipelines

### ISSUE THREAD

**Issue by rgarg1** [openshiftio/openshift.io#1539]
1) Create a space
2) Create new "Vertx. HTTP Booster" project
3) Observe that Build#2 is running.

Observations: 
- Instead of one, two builds were in fact triggered, namely Build#1 and Build#2
- Instead of Build#1 being active, it remains stuck and Build#2 is finished first.
- Cancelling Build#2, makes Build#1 active

**Comment by rgarg1** [comment]
![two builds triggered](https://user-images.githubusercontent.com/28748448/33757566-b96663be-dc20-11e7-8ecf-ccdfa4d81150.png)

The image shows the two builds that were triggered. Ignore Build#3 and #4.

**Comment by sbose78** [comment]
duplicate  of https://github.com/openshiftio/openshift.io/issues/1543
@rgarg1 for all Che/Jenkins(pipeline) related issues, could you please include the version of Jenkins/Che that is running, in the issue description ?
( hope you've done an `update tenant` before starting? :) )

**Comment by jerboaa** [comment]
Looks like a duplicate of  #1355

**Comment by bartoszmajsak** [comment]
Mother of all issues: https://github.com/openshiftio/openshift.io/issues/592

**Comment by qodfathr** [comment]
Closing as dupe.
